Jenkinsfile

```diff
@@ -195,10 +213,11 @@ rm -rf tarball artifacts-*
 chmod +w *.xz
 mkdir -p "artifacts-${suffix}"
 mv -v *.xz "artifacts-${suffix}"
+[ ! -d ftp ] || tar -cJvf "artifacts-${suffix}/cheribsd-ftp-${suffix}.tar.xz" ftp
```

Comment:
Is this the already compressed distribution tarballs?

Reply:
Yeah. I could skip the compression here, but want to at least tar it up, and .tar.xz keeps things consistent.